RIS.js

```diff
@@ -2069,7 +2065,7 @@ var testCases = [
 				"abstractNote": "Abstract",
 				"archive": "Name of Database",
 				"archiveLocation": "Accession Number",
-				"extra": "Publication Number",
+				"extra": "DOI: 10.1234/123456\nPublication Number",
```

Comment:
Sorry, my fault. Somehow I switched the meaning of green and red here in my mind. The doi is here, which is good.